ProjectDplusDsSparse.py

```diff
@@ -35,6 +36,9 @@
                     help='First path of the pT weights file, second name of the pT weights histogram')
 parser.add_argument('--Bspecie', metavar='text', default='all',
                     help='Flag to select b-hadron species for the FD efficiency. Options: (all, B0, B+, Bs, Lb)')
+parser.add_argument('--Bspeciesweights', type=float, nargs=5, required=False,
+                    help='values of weights for the different hadron species '
+                         '(B0weight, Bplusweight, Bsweight, Lbweight, Otherweight)')
```

Comment:
Maybe we can add a default to `[1.,1.,1.,1.,1.]`. Probably we can do this when `--Bspecie` will be removed

Reply:
We can surely add it when we remove `--Bspecie`, but it is not really necessary, because if no values are provided, nothing is applied (and hence are implicitly `[1.,1.,1.,1.,1.]`). If added, we should also change the logic, because at the moment if the argument is not parsed, it is `None` (and I would avoid to apply it in any case, because it really slows down the projection)

Reply:
Ok, but if you call `--Bspeciesweights` what happens? Does the script not start?

Reply:
Yes, it should complain that 5 values are needed. We can test it to double check

Reply:
Ok, fine then.